src/util/postgres_client.js

```diff
@@ -300,6 +340,22 @@ class PostgresTable {
             return res;
         } catch (err) {
             dbg.error(`updateOne failed`, selector, update, query, err);
+            throw err;
+        }
+    }
+
+    async updateOneTransactional(transaction, selector, update, options = {}) {
```

Comment:
I would maybe add `_` to mark it as internal, we don't want to expose this as far as I understand.